.prettierrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "singleQuote": true,
+  "semi": true,
```

Comment:
Blocking: semi should be `false`. We don't need the semicolon at the end of the line.